### ISSUE TITLE
Not working on Android 7, API 25

### ISSUE THREAD

**Issue by vinothm1994** [ybq/Android-SpinKit#38]
this is not working on Android 7, API 25
How I can use on this version of android sdk?

**Comment by RowlandOti** [comment]
Yeap, I think I have a similar problem. It just can't seem to show the ProgressBar. I have setup everything correctly, as in the example. However, the samples, do run correctly on the same device. I tried overriding the default colors to no avail.

**Comment by huywithtravel** [comment]
Have you enabled animation on your device?
I faced with the same issue but made it work after enablind animation on my device

**Comment by RowlandOti** [comment]
Users don't know about enabling animation on the device, hence that is not a solution that will work. In production, you can't ask users to go that deep in the developer settings just for the sake of  a single app.

**Comment by huywithtravel** [comment]
Yes. We can check whether animation is enabled or not on a device by checking below system setting flags:
- WINDOW_ANIMATION_SCALE
- TRANSITION_ANIMATION_SCALE
And due to this system settings, I don't think we can do more further without asking user to enable animation on his/her own device
